### ISSUE TITLE
OneDrive File Picker JavaScript SDK v7.0 - IE issue

### ISSUE THREAD

**Issue by Draggon** [OneDrive/onedrive-api-docs#414]
Hi

I'm using OneDrive File Picker JavaScript SDK v7.0 in may Outlook 365 app (add-in). Everything works fin in chrome but in IE some security settings (on some computers) cause issues and as a result:
- file picker is shown but in browsing mode (no p=2 at the end of the URL = no OK / Cancel buttons)
- after authentication application picker does not redirect to OneDrive

It looks like:
- there is problem with accessing popup window 
- window.name which is used to pass data is cleared during redirect

Changing some security settings like 'Protected mode' or trusted sites seems to solve this - but it's hard to find correct configuration.

Thanks

**Comment by daboxu** [comment]
Hi @Draggon thanks for updating that, right now we do rely on window.name to pass config and for your case in IE, because the browser behavior regard redirect from untrusted site to trusted site is cross boundary so it clears the window.name. Hope this answer your question.

**Comment by Draggon** [comment]
I've found that my issues are also related with using picker in in IFrame with sandbox attribute. Despite of all required parameters IE does not allow to redirect to 'login.microsoftonline.com' in our picker in this scenario.

Finally I've managed to make it working by enabling "Protected mode' for internet / intranet / safe sites and adding 'https://login.microsoftonline.com' and my app url to safe sites.

Honestly I do not understand why 'Protected mode' helps but now everything works fine.
